### PR TITLE
V6

### PR DESCRIPTION
**Pull request by mongjong59**
@justin808 
And please notice that on your [v5 justin808](https://github.com/shakacode/react_on_rails-test-new-redux-generation/pull/8) branch the generator wasn't up-to-date. It's using a previous version with linter issues.

### CODE REVIEW

**Comment by justin808** [comment]
I just tried also! Looks great!

File: client/package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    "babel-runtime": "^6.6.1",
+    "babel-polyfill": "^6.7.4",
+    "babel-preset-es2015": "^6.6.0",
+    "babel-preset-react": "^6.5.0",
+    "babel-preset-stage-0": "^6.5.0",
+    "es5-shim": "^4.5.7",
+    "expose-loader": "^0.7.1",
+    "imports-loader": "^0.6.5",
+    "react": "^0.14.8 || ^15.0.0",
+    "react-dom": "^0.14.8 || ^15.0.0",
```
Comment (justin808) [inline client/package.json:32]: @nostophilia Here's that 0.14.8 again.

File: client/package.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "react-webpack-rails-tutorial",
```
Comment (justin808) [inline client/package.json:2]: @nostophilia: We get these warnings:

```
npm WARN react-webpack-rails-tutorial@0.0.1 No description
npm WARN react-webpack-rails-tutorial@0.0.1 No repository field.
npm WARN react-webpack-rails-tutorial@0.0.1 No license field.
npm WARN react-webpack-rails-tutorial@0.0.1 No repository field.
npm WARN react-webpack-rails-tutorial@0.0.1 No license field.
```

File: package.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "react-webpack-rails-tutorial",
```
Comment (justin808) [inline package.json:2]: @nostophilia: We get these warnings:

```
npm WARN react-webpack-rails-tutorial@0.0.1 No description
npm WARN react-webpack-rails-tutorial@0.0.1 No repository field.
npm WARN react-webpack-rails-tutorial@0.0.1 No license field.
npm WARN react-webpack-rails-tutorial@0.0.1 No repository field.
npm WARN react-webpack-rails-tutorial@0.0.1 No license field.
```

File: app/views/hello_world/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<h1 class="alert alert-info this-works">Hello World</h1>
```
Comment (justin808) [inline app/views/hello_world/index.html.erb:1]: @nostophilia 

There should be no class values on this line.

```
<h1>Hello World</h1>
```

File: client/package.json (hunk 2 above)
Comment (mongjong59) [reply client/package.json:2]: I've added `"private": true,` under 

```
"name": "react-webpack-rails-tutorial",
"version": "0.0.1",
```

pushed v7

**Pull request closed without merging**